.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,33 @@
+name: NPM Release
+
+on:
+  release:
+    types: [created]
```

Comment:
What's about an update? For example, if someone reinitiates a tag, for example, because of a human error? Will this rule reupload the package?

Reply:
Good point!!!
I will change trigger "create"  on "published"

Reply:
Another point is that we can't release the module with the same version in package.json
Error example:

> error Couldn't publish package: "https://registry.npmjs.org/xxx: You cannot publish over the previously published versions: x.x.x."